### PR TITLE
Removed URSA and added my own PemKey generation without OpenSSL insid…

### PR DESCRIPTION
**Pull request by brandwe**
I have removed URSA and added my own Pemkey generation without depending on OpenSSL.

### CODE REVIEW

File: lib/passport-azure-ad/aadutils.js [hunk 1]
```diff
@@ -114,3 +119,66 @@ exports.uid = function(len) {
...
+    var modulus_hex = modulus.toString('hex')
+    var exponent_hex = exponent.toString('hex')
+
+    modulus_hex = prepadSigned(modulus_hex)
+    exponent_hex = prepadSigned(exponent_hex)
+
+    var modlen = modulus_hex.length/2
+    var explen = exponent_hex.length/2
+
+    var encoded_modlen = encodeLengthHex(modlen)
```
Comment (weijjia) [inline lib/passport-azure-ad/aadutils.js]: are you missing the ; here?

Comment (weijjia) [inline lib/passport-azure-ad/aadutils.js]: actually you're missing ';' in a couple of places.

**Comment by weijjia** [comment]
looks good except some format issues.

**Comment by brandwe** [comment]
I have fixed the formatting issues you referenced.

**Pull request merged**